src/test/scala/edu/ie3/mobsim/model/TripSimulationData.scala

```diff
@@ -0,0 +1,76 @@
+/*
+ * © 2022. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+ */
+
+package edu.ie3.mobsim.model
+
+import edu.ie3.mobsim.io.geodata.{
+  PoiEnums,
+  PoiTestData,
+  PoiUtils,
+  PointOfInterest
+}
+import edu.ie3.mobsim.io.probabilities.{
+  DrivingSpeed,
+  ProbabilityDensityFunction
+}
+import edu.ie3.mobsim.io.probabilities.DrivingSpeed.SpeedFunction
+import edu.ie3.util.quantities.PowerSystemUnits
+import tech.units.indriya.ComparableQuantity
+import tech.units.indriya.quantity.Quantities
+import tech.units.indriya.unit.Units.KILOMETRE_PER_HOUR
+
+import javax.measure.quantity.Energy
+
+trait TripSimulationData extends ElectricVehicleTestData with PoiTestData {
+  val ev: ElectricVehicle = ElectricVehicle.buildEv(
+    "test_car",
+    givenModel,
+    givenHomePoi,
+    givenWorkPoi,
+    givenSimulationStart,
+    givenFirstDeparture,
+    true
```

Comment:
Please name boolean parameter

```suggestion
    isChargingAtHomePossible = true
```